solver/src/main.rs

```diff
@@ -183,7 +183,7 @@ struct Arguments {
     /// network before going to back to solving.
     #[structopt(
         long,
-        default_value = "90",
+        default_value = "120",
```

Comment:
Not that I disagree with the change, just wondering the rational behind it.

Reply:
:see_no_evil: Its in the description: 
> Increased deadline for flashbots transaction submitting from 90s to 120s (with flashbots occasional slow API this should reduce number of deadline expires)

Reply:
I noticed that we had several deadline expirations and right after our tx was mined. This is not that important change, we can leave it or remove it (since we can change it as being a config parameter), but I felt that 120 is more adequate.

Reply:
No its fine, I just missed it in the description. My bad!

Reply:
It would be great if - once we clean this up - we can unify this strategy with Archer and our default submission strategy (this might mean to change this to number of blocks rather than time)